pegasus/helper_modules/dashboard.rb

```diff
@@ -14,6 +14,11 @@ def self.admin?(user_id)
     !!db[:users][id: user_id, admin: true]
   end
 
+  def self.ai_api_access?(user_id)
```

Comment:
Do we need this (this seems like it would be relevant on pegasus pages, are we using these APIs there)?

Reply:
That makes sense -  Will remove. Not as familiar with backend so thank you!